File: 프로그래머스/0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.py
def solution(numbers):
    max_num = -100000000
    if len(set(numbers)) == 1: return numbers[0] * numbers[1]
    for i in numbers:
        for j in numbers:
            if numbers.index(j) == numbers.index(i): continue
            if i * j >= max_num:
                max_num = i*j
    
    return max_num

File: 프로그래머스/0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.swift
import Foundation

func solution(_ numbers:[Int]) -> Int {
    var rst = numbers[0] * numbers[1]
     
    for i in 0 ..< numbers.count {
        for j in 0 ..< numbers.count {
            if i == j {
                continue
            }
            
            let tmp = numbers[i] * numbers[j]
            if  tmp >= rst {
                rst = tmp
            }
        }
    }
    
    return rst
}